playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts

```diff
@@ -45,7 +45,7 @@ test.describe('Adult Dependent visual tests', () => {
     // Clear the error message by select `The dependant being enrolled`
     await whoHasVentricleHeartDefect.uncheck(WHO.SomeoneElse);
     await whoHasVentricleHeartDefect.check(WHO.TheDependantBeingEnrolled);
-    await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
+    await expect(whoHasVentricleHeartDefect.errorMessage()).toBeHidden();
```

Comment:
Use assertion `expect` to wait for UI changes. Assertions can help users understand easily why it fails here.